web/docs/migrate-from-0-11-to-0-12.md

```diff
@@ -71,7 +71,7 @@ projects.
 
 In Wasp 0.11.X, authentication was based on the `User` model which the developer needed to set up properly and take care of the auth fields like `email` or `password`.
 
-```wasp title="main.wasp"
+```wasp title="main.wasp" {18-31}
```

Comment:
I highlighted the fields that get removed with new auth.

Reply:
Might I suggest using `// highlight-next-line` and `// highlight-start` and `// highlight-end` comments. They are more resistant to code changes in the future :) 

https://docusaurus.io/docs/markdown-features/code-blocks#highlighting-with-comments

Reply:
You might indeed :D! Oh yes this is great, I learned about highlight-next-line, but I forgot again about start/end, that is awesome.